### ISSUE TITLE
Messages by email and web

### ISSUE THREAD

**Issue by jdavidb** [SoylentNews/rehash#285]
Message notifications right now can be set to "none," "email," or "web."  It would be nice to have "email and web" as an option.

**Comment by TheMightyBuzzard** [comment]
I don't disagree but this is a more significant change than it looks like it would be at first glance. It won't be making it into the next update (very soon) but possibly the one afterwards.

**Comment by jdavidb** [comment]
Makes sense.  I know what it's like sometimes where to an outside observer
something seems like you could just do it and the developer knows that it
really requires a descent into the ninth circle of hell to fight the code
and make it possible. :)  It's just a "nice to have" that occurred to me.

You guys are awesome, btw.

**Comment by TheMightyBuzzard** [comment]
Oh it's not a ninth circle thing. Just not a five minute thing either. Probably take a morning to write up but then it needs testing time on dev as well and we're in feature freeze for the next release aside from a couple things that we've been putting off finishing.

**Comment by Open4D** [comment]
This preference is set at https://soylentnews.org/my/messages

While you are changing that screen anyway, I would propose some moderate changes to the format/layout, as per the attached PDF. (Github doesn't accept ODS files here, but I can provide the original on request.)
[messagePrefs-v03.pdf](https://github.com/SoylentNews/rehash/files/454938/messagePrefs-v03.pdf)

The main change I'm proposing that is enabled by the jdavidb's feature request, is the use of 2 columns of checkboxes ("Web" and "Email").  But I have also added a "Notes" column which includes some hyperlinks.  And I've tried to make the final preference (comment thresholds) more distinct, partly by reordering the other preferences.
